src/_vars.scss

```diff
@@ -10,6 +10,8 @@
 $primary-blue: #014983;
 $primary-cyan: #00aeef;
 
+$primary-red: #ff0000;
```

Comment:
I think this variable needs to be renamed. The `primary` and `secondary` names refer specifically to [Gordon's brand design standards](https://gordonedu.sharepoint.com/WebLinks/Shared%20Documents/Design%20Center/Gordon%20Colors.pdf?originalPath=aHR0cHM6Ly9nb3Jkb25lZHUuc2hhcmVwb2ludC5jb20vOmI6L2cvV2ViTGlua3MvRWFyQVZ1YmM2Y3hNcGRRNFBFMXM0YXNCRUxtTFgwVUFHU0tLWmI4MHZfTVp6QT9ydGltZT1HLWVqQUx2bzJFZw), and so to call this `primary-red` is misleading because it isn't one of our primary brand colors.

Reply:
Ahhh..! I did not know that at all! Thanks! :)